keyboards/idb/idb_60/idb_60.c

```diff
@@ -5,15 +5,26 @@ void keyboard_pre_init_kb(void) {
     setPinOutput(C5);
 }
 
-void led_set_kb(uint8_t usb_led) {
+// Capslock LED indicator
 
+void led_set_kb(uint8_t usb_led) {
     if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
-		_idb_60_caps_led_on();
-		} else {
-		_idb_60_caps_led_off();
+        _idb_60_caps_led_on();
+    } else {
+        _idb_60_caps_led_off();
     }
+    led_set_user(usb_led);
+}
 
-	led_set_user(usb_led);
+// Layer LED indicator - drive LED when not on base layer
+
+layer_state_t layer_state_set_kb(layer_state_t state) {
```

Comment:
Having `layer_state_set_kb` here with fixed behaviour makes it difficult for users to override. Either,
1. Move it back to the keymap level
2. have a weak layer_state_set_user here instead

Reply:
Oh. I figured it'd follow the cascading scheme of most other things in QMK. [This part of the docs](https://beta.docs.qmk.fm/using-qmk/guides/custom_quantum_functions#layer_state_set_-function-documentation) could perhaps be cleared up then or somehow elaborate on that.

I intended to provide a default behaviour that can be overridden on the keymap level. As such, I suppose going for a `layer_state_set_user` callback on the keyboard level is the move for that? Feels kind of wrong to me, but I'll be happy with whatever makes sense in the grand scheme of things.

Reply:
As long as the implementation is weak, that's fine. That way a user can still provide their own implementation if they so choose.

Reply:
Alright, donezo!